WisangAnggoro/fuzzy_api
# Fix json parameters and Add Gunicorn

# Changes
- Fix json parameter
- Fix json response
- Update readme
- Add wsgi server

# Test WSGI
- make sure venv is activated
![image](https://github.com/WisangAnggoro/fuzzy_api/assets/19965509/d9a21e97-d83b-4a02-b802-3bb7cb2ff22a)
- install pip dependency
```bash
pip install -r requirements.txt 
```
- Run the server
```bash
gunicorn --bind 0.0.0.0:5000 wsgi:app
```
